art/attacks/fast_gradient.py

```diff
@@ -33,9 +33,11 @@ class FastGradientMethod(Attack):
     Gradient Sign Method"). This implementation extends the attack to other norms, and is therefore called the Fast
     Gradient Method. Paper link: https://arxiv.org/abs/1412.6572
     """
-    attack_params = Attack.attack_params + ['norm', 'eps', 'targeted', 'num_random_init', 'batch_size']
+    attack_params = Attack.attack_params + ['norm', 'eps', 'targeted', 'num_random_init', 'batch_size', 'minimal',
+                                            'eps_max', 'eps_step']
 
-    def __init__(self, classifier, norm=np.inf, eps=.3, targeted=False, num_random_init=0, batch_size=128):
+    def __init__(self, classifier, norm=np.inf, eps=.3, targeted=False, num_random_init=0, batch_size=128,
+                 minimal=False, eps_max=1.0, eps_step=0.1):
```

Comment:
I suggest we remove the `eps_max` parameter for the minimal case and use `eps` instead? The two parameters mean the same thing, they were just separated before because the two cases were treated apart.

Reply:
Yes, I think that's a good idea

Reply:
resolved with fad11ed6d14bb9277f4988e62a9ffecd05a82491